scripts/local_rabbitmq.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+RABBITMQ_PID_FILE=/var/lib/rabbitmq/pid
+RABBITMQ_DEFAULT_USER=admin
+RABBITMQ_DEFAULT_PASS=banana
+RABBITMQ_DEFAULT_VHOST=/
+MAX_WAIT_SECONDS=30
+RABBITMQ_LEAGUE_SERVICE_USER=faf-league-service
+RABBITMQ_LEAGUE_SERVICE_PASS=banana
+RABBITMQ_LEAGUE_SERVICE_VHOST=/faf-lobby
+
+docker run -d -p 5672:5672 --restart unless-stopped --name faf-rabbitmq rabbitmq:3.8.2-management-alpine
```

Comment:
Is it necessary to specify the bugfix version?
```suggestion
docker run -d -p 5672:5672 --restart unless-stopped --name faf-rabbitmq rabbitmq:3.8-management-alpine
```

Reply:
I guess it is not needed

Reply:
I went ahead and also removed the minor version, because test.yml also uses the latest minor version

Reply:
Just be careful because I had issues with 3.9 on the lobby server so I had to pin it at 3.8.

Reply:
yeah, I should probably use 3.8 like also defined in faf stack. no use if it works locally but breaks when it is actually deployed :D